hlx_statics/components/divider.js

```diff
@@ -0,0 +1,15 @@
+export default function createDivider(size, style, orientation = 'horizontal') {
```

Comment:
EDS equivalent of Gatsby [Divider](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Divider/index.js#L18) component